test/specs/addons/Confirm-test.js

```diff
@@ -7,29 +7,27 @@ describe('Confirm', () => {
     render(<Confirm />).first().props.abortLabel.should.equal('Cancel');
   });
   it('default prop confirmLabel should be "Yes"', () => {
-    render(<Confirm />).first().props.confirmLabel.should.equal('Yes');
+    Confirm.defaultProps.confirmLabel.should.equal('Yes');
   });
   it('default prop ref should be "modal"', () => {
-    render(<Confirm />).first().props.ref.should.equal('modal');
+    Confirm.defaultProps.ref.should.equal('modal');
```

Comment:
These tests were testing that defaultProps was working, which is React's responsibility.  Now we are testing that defaultProps is set as the description says.